core/flyout_horizontal.ts

```diff
@@ -309,18 +317,18 @@ export class HorizontalFlyout extends Flyout {
     }
 
     const flyoutRect = this.svgGroup_.getBoundingClientRect();
-    // BIG_NUM is offscreen padding so that blocks dragged beyond the shown
+    // bigNum is offscreen padding so that blocks dragged beyond the shown
     // flyout area are still deleted.  Must be larger than the largest screen
     // size, but be smaller than half Number.MAX_SAFE_INTEGER (not available on
     // IE).
-    const BIG_NUM = 1000000000;
+    const bigNum = 1000000000;
```

Comment:
did you change this because the linter doesn't allow constant case in samples? if so, we should probably update that...

on the other hand, this doesn't meet the [requirements](https://google.github.io/styleguide/tsguide.html#identifiers-constants) of using constant case so it should be lower case anyway

Reply:
I changed it because it doesn't meet the requirements and it's been bugging me.

Reply:
cool, makes sense. i had to go look up the rules because i was worried the linter wasn't allowing it